src/UserFrontend.php

```diff
@@ -60,8 +60,20 @@ public static function login_head() {
    * @implements login_form_defaults
    */
   public static function login_form_defaults(array $args) {
+    if (file_exists(get_stylesheet_directory() . '/dist/styles/login.css')) {
+      wp_enqueue_style('theme/login', get_stylesheet_directory_uri() . '/dist/styles/login.css');
+    }
+    else {
+      echo <<<EOD
+<style>
+#loginform .login-remember {
+  display: none;
```

Comment:
This happens in line 105 already: https://github.com/netzstrategen/wordpress-core-standards/pull/9/files#diff-a4a15f99f66b9d0b641ea2f50c52c379R105

Reply:
The selector is different because on line 105 applies on the login page.